.travis.yml

```diff
@@ -4,11 +4,15 @@ sudo: false
 python:
   - 2.7
   - 3.6
+env:
+  pip_schema=full
+  pip_schema=core
```

Comment:
Hmm, I'm a bit confused about why this works.  Isn't that just setting
the environmental variable `pip_schema` to "full" and then to "core"
right after in the same session?  Looking at the logs, I see just two
runs and

```
434 $ export pip_schema=full
435 $ export pip_schema=core
```

Reply:
no -- it builds a matrix -- now each python version and this setting combine into 4 runs (all possible combinations)

Reply:
So, under #31 packages structure, the lines starting with [this
line][0] wouldn't get executed, I think.  Your change would end up
reducing the coverage because it doesn't actually test the jsonschema
case (again, I think).

[0]: https://github.com/pyout/pyout/blob/824cc1a6d5a6bdc3dc306424c7eeeb8fbfbe3b97/pyout/styling.py#L158

Reply:
> no -- it builds a matrix -- now each python version and this setting combine into 4 runs (all possible combinations)

That's the intention, yeah.  Did you check the result?

Reply:
I see two runs: https://travis-ci.org/pyout/pyout/builds/333902131

Reply:
Perhaps you want `env` under a `matrix`?

Reply:
Now I am confused as well in how I got to 100% 
Probably that one of the pythons comes with the module already.. yeah, I will fix it in 5 minutes. Was mislead by some incorrect example I guess

Reply:
https://docs.travis-ci.com/user/build-stages/matrix-expansion/

Reply:
Ah I did not add - to make it into a list

Reply:
> Now I am confused as well in how I got to 100%

The current code (unlike what's in #31) just does

https://github.com/pyout/pyout/blob/7abce8355c63dbcbfcc9ee5d30f00d9fa18778f9/pyout.py#L18-L21

so even with the exception, all lines are executed.